CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ else()
   set(BENCHMARK_BUILD_32_BITS OFF CACHE BOOL "Build a 32 bit version of the library - unsupported when using MSVC)" FORCE)
 endif()
 option(BENCHMARK_ENABLE_INSTALL "Enable installation of benchmark. (Projects embedding benchmark may want to turn this OFF.)" ON)
+option(BENCHMARK_ENABLE_DOCS_INSTALL "Enable installation of documentation." ON)
```

Comment:
I would've suggested `BENCHMARK_INSTALL_DOCS`, but then there's already `BENCHMARK_ENABLE_INSTALL`, so i guess this is fine.

Reply:
Sure. Fixed.

Reply:
(Note how i worded that, i wasn't asking to change the diff because it was consistent)

@dominichamon do we want to be consistent with the existing options, or have maybe better(worse?) new options?